### DISCUSSION TITLE
Doublevpn

### DISCUSSION THREAD

**Discussion by ghost** [Zackptg5/Wireguard-Pi-Hole-Cloudflared-Unbound-DNSCrypt-VPN-Server#10] [Q&A] [answered]
Hello Zack, thank you for your script :) Everything works smoothly without any errors. 
Can you please add instructions on how to use Double VPN with your script?
Thanks in advance.

**Comment by Zackptg5** [comment, marked as answer]
You'd need to run this script on each node, and then modify the wireguard server/client files as needed for it. I won't be adding anything specific for it here since it's not something I'd use (or see the point of - just adds extra latency)
